src/styles/app.scss

```diff
@@ -5,8 +5,17 @@ html, body, #root {
   width: 100%;
 }
 
+#root {
+  overflow-x: hidden;
```

Comment:
Ah, I see this is where you moved the changes I referenced a few comments earlier :)